src/lakefs_spec/client_helpers.py

```diff
@@ -25,3 +27,33 @@ def commit(
     commit_creation = CommitCreation(message=message, metadata=metadata or {})
 
     client.commits_api.commit(repository=repository, branch=branch, commit_creation=commit_creation)
+
+
+def create_tag(client: LakeFSClient, repository: str, ref: str, tag: str) -> None:
+    tag_creation = TagCreation(tag, ref=ref)
+    client.tags_api.create_tag(repository=repository, tag_creation=tag_creation)
+
+
+def get_tags_and_commits(client: LakeFSClient, repository: str) -> dict:
+    return client.tags_api.list_tags(repository=repository)
+
+
+def merge(client: LakeFSClient, repository: str, source_ref: str, target_branch: str) -> None:
+    diff = client.refs_api.diff_refs(
+        repository=repository, left_ref=target_branch, right_ref=source_ref
+    )
+    if not diff.results:
+        logger.warning("No difference between source and target. Aborting merge.")
+        return
+    client.refs_api.merge_into_branch(
+        repository=repository, source_ref=source_ref, destination_branch=target_branch
+    )
+
+
+def revert(
+    client: LakeFSClient, repository: str, branch: str, revert_to_parent_number: int = 1
```

Comment:
Could you rename the argument to `parent_number`, and then write a small docstring saying something like "Revert a branch by ``parent_number`` commits"?